website/docs/docs/deploy/deploy-jobs.md

```diff
@@ -37,8 +37,8 @@ You can create a deploy job and configure it to run on [scheduled days and times
     - **Run source freshness** &mdash; Enable this option to invoke the `dbt source freshness` command before running the deploy job. Refer to [Source freshness](/docs/deploy/source-freshness) for more details.
 4. Options in the **Triggers** section:
     - **Run on schedule** &mdash; Run the deploy job on a set schedule.
-        - **Timing** &mdash; Specify whether to [schedule](#schedule-days) the deploy job using **Hours of the day** that runs the job at specific times of day, **Exact intervals** that runs the job every specified number of hours, or **Cron schedule** that runs the job specified using [cron syntax](#cron-schedule).
-        - **Days of the week** &mdash; By default, it’s set to every day when **Hours of the day** or **Exact intervals** is chosen for **Timing**.
+        - **Timing** &mdash; Specify whether to [schedule](#schedule-days) the deploy job using **Intervals** that runs the job every specified number of hours, **Specific hours** that runs the job at specific times of day, or **Cron schedule** that runs the job specified using [cron syntax](#cron-schedule).
```

Comment:
There's so much bold on this page, I wonder if switching the in-paragraph optiong to italics might make them easier to read? 
```suggestion
        - **Timing** &mdash; Specify whether to [schedule](#schedule-days) the deploy job using *Intervals* that run the job every specified number of hours, *Specific hours* that run the job at specific times of day, or *Cron schedule* that runs the job specified using [cron syntax](#cron-schedule).
```

Reply:
yeah, that's fair. i totally see what you're saying.

iirc, italics are also not great (and should be used sparingly) since they're not great for people with disabilities (the slant makes it more challenging to read) so 🤷‍♀️ i'll leave as is for now but gonna take your "runs > run" catch!